### ISSUE TITLE
[Feature request] Custom fast server switch (temporary change to server not in current filter/selection)

### ISSUE THREAD

**Issue by bebemls** [mullvad/mullvadvpn-app#4244]
## Issue report

Operating system: macOS Ventura (not specific to this OS)

App version: 2022.5

### Issue description

Some times I need change the server to one that is not included in the filter, and it would be so much easier if was a possibility to have that option.

<p align="center"><img src="https://i.imgur.com/5Ueo0aG.png" width=30% height=30%></p>

Something like this. It would be much easier to switch to a specific server. In my case a fast switch to the country where I live.
Basically a fast server switch, or a favourite servers list.

**Comment by faern** [comment]
This should essentially be solved with the "custom lists" feature. That feature is being tracked over at #3180. We'll note that there is a use case where the filters should then be per custom list, and not global.

I'm going to close this in favor of tracking everything related to custom lists in the other issue.
